tests/test_gql_utils.py

```diff
@@ -0,0 +1,35 @@
+from origin_sdk.gql.queries.input_queries.session import (
+    create_get_session_gql,
+    tree_to_string,
+    RecursiveTree,
+)
+
+test_object: RecursiveTree = {
+    "a": None,
+    "b": None,
+    "c": "ShouldNotTree",
+    "d": ["Could", "Tree", "But", "shouldn't"],
+    "e": {"Will": True, "For": True, "Sure": True, "Tree": False},
+    "getTechnology (technologyName: $technologyName)": {"name": None},
+}
```

Comment:
Here are the tests for the `tree_to_string` functionality we were talking about this morning.